src/definitions/behaviors/form.js

```diff
@@ -1388,7 +1390,22 @@
                                     var invalidFields = module.validate.rule(field, rule, true) || [];
                                     if (invalidFields.length > 0) {
                                         module.debug('Field is invalid', identifier, rule.type);
-                                        fieldErrors.push(module.get.prompt(rule, field));
+                                        var fieldError = module.get.prompt(rule, field);
+                                        if (!settings.inline) {
+                                            if (
+                                                // Always allow the first error prompt for new field identifiers
+                                                (!(identifier in formErrorsTracker)
+                                                // Also allow multiple error prompts per field identifier but make sure each prompt is unique
+                                                || formErrorsTracker[identifier].indexOf(fieldError) === -1)
+                                                // Limit the number of unique error prompts for every field identifier if specified
+                                                && (!errorLimit || formErrorsTracker[identifier].length < errorLimit)
```

Comment:
In case error limit is > 0 and also the identifier wasn't set yet, the .length access triggers 'cannot read .length from undefined'

```suggestion
                                                && (!errorLimit || (formErrorsTracker[identifier] || []).length < errorLimit)
```

Reply:
Done. See commit db538f5